examples/22_define-router/src/entries.tsx

```diff
@@ -0,0 +1,156 @@
+import { new_defineRouter } from 'waku/router/server';
+import { Slot, Children } from 'waku/client';
```

Comment:
is `waku/client` the best place for these? client to me would mean these are targeted towards the scope of client components, but they're just used here for the router to be able to target where to replace with each level of rendering components (regardless of client vs server)

that is if I understand correctly

Reply:
They are client components. We can embed client components in a server component. It's confusing, but it's how it works. createPages API will hide this complexity anyway.